json-simple/src/test/java/org/json/simple/Test.java

```diff
@@ -302,7 +302,7 @@ public void testEncode() throws Exception{
 		array1.add("abc\u0010a/");
 		array1.add(new Integer(123));
 		array1.add(new Double(222.123));
-		array1.add(new Boolean(true));
+		array1.add(Boolean.valueOf(true));
```

Comment:
Cannot simply write "array1.add(true);" here?
Or if not, it would be better to create a static constant to boolean value to use this, like:

public static boolen BOOLEAN_TRUE = Boolean.valueOf(true);
public static boolen BOOLEAN_FALSE = Boolean.valueOf(false);